app/index.tsx

```diff
@@ -1,4 +1,7 @@
+// eslint-disable-next-line import/no-unresolved
+import "vite/modulepreload-polyfill";
```

Comment:
It’s recommended to add this when integrating with a backend: https://vitejs.dev/guide/backend-integration.html#backend-integration